static/docs/commands-reference/commit.md

```diff
@@ -70,11 +70,9 @@ It handles that last step of adding the file to the DVC cache.
   backward through the pipeline from the named target(s). This means DVC will
   not commit files referenced later in the pipeline than the named target(s).
 
-- `-R`, `--recursive` - the `targets` value is expected to be a directory path.
-  With this option, `dvc commit` determines the files to commit by searching the
-  named directory, and its subdirectories, for DVC-files for which to commit
-  data. Along with providing a `target`, or `target` along with `--with-deps`,
-  it is yet another way to limit the scope of DVC-files to upload.
+- `-R`, `--recursive` - determines the files to commit cache- by searching the
```

Comment:
`commit cache-`?

Reply:
Guys, note that I've already made several changes to the `--recursive` option explanation in my recent commits, currently under review in PR #441 so depending which one gets merged first, will require merging. Probably my PR will be merged first so please see if those changes are good enough in which case we can remove the changes to the same text done in this PR. Thanks

Reply:
Never mind! I was thinking about the `--with-deps` option! Carry on 😅

Reply:
Agree with Ivan: remove `cache- `, seems like a typo?

Reply:
Wait... Me again, SORRY.

Actually I DID update this option already. (I had forgotten to push the commit to my PR until now). See for example https://github.com/iterative/dvc.org/pull/441/files#diff-cf1201ba4e234c56d863d0adad07fc7aR82

Reply:
@jorgeorpinel have you also added -R to repro? That was the original scope of this issue.

Reply:
seems like you have not added -R option in `dvc commit`

Reply:
I had not until now. Added the change o my newest PR now (See https://github.com/iterative/dvc.org/pull/456/files#diff-3bde69637284b09b8cfc78280e173e7a)